addons/ch53_fix/$PBOPREFIX$

```diff
@@ -0,0 +1 @@
+z\TF47\addons\ch53_fix
```

Comment:
könnten wir das module bzw. addon in cup_ch53_patch umbennen?

Reply:
Ich entfern das nochmal, war eig für RHS gedacht